cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -77,7 +77,7 @@ func NewCMServer() *CMServer {
 		NodeMilliCPU:            1000,
 		NodeMemory:              resource.MustParse("3Gi"),
 		SyncNodeList:            true,
-		SyncNodeStatus:          false,
+		SyncNodeStatus:          true,
```

Comment:
why change the default value?  we are not syncing node status in controller manager now.